go.mod

```diff
@@ -3,20 +3,54 @@ module sigs.k8s.io/aws-iam-authenticator
 go 1.16
 
 require (
-	github.com/aws/aws-sdk-go v1.43.28
-	github.com/gofrs/flock v0.7.0
-	github.com/manifoldco/promptui v0.9.0
-	github.com/prometheus/client_golang v1.11.0
-	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/cobra v1.1.3
-	github.com/spf13/viper v1.7.0
-	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
-	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.22.1
-	k8s.io/apimachinery v0.22.1
-	k8s.io/client-go v0.22.1
-	k8s.io/code-generator v0.22.1
-	k8s.io/component-base v0.22.1
-	k8s.io/sample-controller v0.22.1
-	sigs.k8s.io/yaml v1.2.0
+    github.com/aws/aws-sdk-go v1.44.107
+    github.com/gofrs/flock v0.7.0
+    github.com/manifoldco/promptui v0.9.0
+    github.com/onsi/ginkgo v1.14.0
+    github.com/onsi/gomega v1.10.1
+    github.com/prometheus/client_golang v1.11.0
+    github.com/sirupsen/logrus v1.8.1
+    github.com/spf13/cobra v1.1.3
+    github.com/spf13/viper v1.7.0
+    golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
+    gopkg.in/yaml.v2 v2.4.0
+    k8s.io/api v0.22.1
+    k8s.io/apimachinery v0.22.1
+    k8s.io/client-go v0.22.1
+    k8s.io/code-generator v0.22.1
+    k8s.io/component-base v0.22.1
+    k8s.io/kubernetes v1.22.0
+    k8s.io/sample-controller v0.22.1
+    sigs.k8s.io/yaml v1.2.0
+)
+
+replace (
```

Comment:
Why are all these replace directives being added?

Reply:
I wasn't paying attention when I was cherry-picking and resolving commits. They were added in the main branch when basic e2e tests were added.